physics/module_bl_mynn.F90

```diff
@@ -1,5 +1,5 @@
 !>\file module_bl_mynn.F90
-!! This file contains the entity of MYNN-EDMF PBL scheme.
+! This file contains the entity of MYNN-EDMF PBL scheme.
```

Comment:
Just to be sure, was the removal of the second `!` intentional? Its removal means that Doxygen will not parse this line anymore. Not that the file description is all that useful in this case (and `entity` should probably be `entirety`), but still.

Reply:
Inadvertent. I'll add it back.